charts/rasa-x/templates/nginx-deployment.yaml

```diff
@@ -78,8 +78,10 @@ spec:
           value: "{{ include "rasa-x.host" . }}.{{ .Release.Namespace }}.svc:{{ .Values.rasax.port }}"
         - name: "RASA_PRODUCTION_HOST"
           value: "{{ include "rasa-x.fullname" . }}-{{ .Values.rasa.versions.rasaProduction.serviceName }}.{{ .Release.Namespace }}.svc:{{ .Values.rasa.port }}"
+        {{- if .Values.app.install }}
         - name: "CUSTOM_ACTION_HOST"
```

Comment:
It looks like we don't use it (`CUSTOM_ACTION_HOST`) anymore, @melindaloubser1 am I right?

Reply:
I see this in the changelog from way back:
```
## [0.20.1] - 2019-08-12

### Features

* nginx upstream hosts can be set using `RASA_X_HOST` (default: `rasa-x:5002`),
  `CUSTOM_ACTION_HOST` (default: `app:5055`), `RASA_PRODUCTION_HOST`
  (default: `rasa-production:5005`)
 
```
So it was at some point used in docker-compose to specify the action server host.
Otherwise no mention of it.

Reply:
Cool, @ferenc-hechler Can you remove the `CUSTOM_ACTION_HOST` variable from the template?

Reply:
Hi @tczekajlo , just updated the issue-174 branch in my repo. I removed the "CUSTOM_ACTION_HOST" environment variable in nginx-deployment and fixed the typo of "existingUrl" in values.yaml.